cloudinary/apiCaller.js

```diff
@@ -37,44 +37,52 @@ const addServerInfo = (imageList, batchSize, dpr, metaData, quality, cb, rollBar
 
 const sendToCloudinary = (imagesArray, batchSize, dpr, metaData, quality, cb, rollBarMsg) => {
   let analyzeResults = [];
+  let uploadErrors = [];
   let timestamp = Math.floor(Date.now() / 1000);
   async.eachLimit(imagesArray, batchSize, (image, callback) => {
-    let context = {
-      rendered_dimensions: {width: image.width, height: image.height},
-      dpr: dpr,
-      request_headers: metaData.headers
-    };
-    let transformations = config.get('cloudinary.transformations', null);
-    if (typeof transformations === 'string') {
-      transformations = JSON.parse(transformations);
-    }
-    let eager = [];
-    if (transformations) {
-      eager = transformations.map((trans) => {
-        let t = Object.assign({}, trans, {width: image.width, height: image.height, dpr: dpr || 1});
-        if (quality) {
-           Object.assign(t, {quality: 'auto:' + quality})
-        }
-        return t;
-      });
-    }
-    const tags = [timestamp];
-    if (image.url === metaData.lcpURL) {
-      tags.push('lcp');
-    }
-
-    cloudinary.v2.uploader.upload(image.url, {eager: eager, analyze:{context: context}, tags: tags},(error, result) => {
-      if (error) {
-        analyzeResults.push({public_id: null});
-        log.error('Error uploading to cloudinary', error, rollBarMsg);
-        callback(error);
-      } else {
-        result.server = image.server;
-        analyzeResults.push(result);
-        callback();
+    if (image.width > 0 && image.height > 0) {
```

Comment:
Perhaps prefer early return if invalid?
```javascript
if (image.width <= 0 && image.height <= 0) {
  return
}
```

Reply:
Is there an actual need to verify this here as well, as you already seem to filter out bad images here:
https://github.com/cloudinary/web-speed-test-server/pull/31/files#diff-b98ce95ff5acc1c70bf9299c7f8c3e764da169496aef4a6bab2bf735df1f1c3fR36

Reply:
you are right, removed it.